sdk/clientcore/core/src/main/java/io/clientcore/core/instrumentation/metrics/Meter.java

```diff
@@ -46,13 +50,14 @@ public interface Meter extends AutoCloseable {
      * </pre>
      * <!-- end io.clientcore.core.instrumentation.histogram -->
      *
-     * @param name short histogram name following <a href="https://opentelemetry.io/docs/specs/semconv/general/naming/">naming conventions</a>
-     * @param description free-form text describing the instrument
-     * @param unit optional unit of measurement following <a href="https://opentelemetry.io/docs/specs/semconv/general/metrics/#units">units conventions</a>
+     * @param name short histogram name following <a href="https://opentelemetry.io/docs/specs/semconv/general/naming/">naming conventions</a>. Required
+     * @param description free-form text describing the instrument. Required
+     * @param unit optional unit of measurement following <a href="https://opentelemetry.io/docs/specs/semconv/general/metrics/#units">units conventions</a>. Required
+     * @param bucketBoundaries list of bucket boundaries for the histogram. Optional
      * @return new instance of {@link DoubleHistogram}
      * @throws NullPointerException if name or description is null.
      */
-    DoubleHistogram createDoubleHistogram(String name, String description, String unit);
+    DoubleHistogram createDoubleHistogram(String name, String description, String unit, List<Double> bucketBoundaries);
```

Comment:
Is `bucketBoundaries` always necessary?

Reply:
It's necessary in most cases to have proper buckets. But it's optional from the API perspective (you can pass null). 
It might become unnecessary in the future (with dynamically adjusted boundaries on OTel side).